Tests/Unit/EmogrifierTest.php

```diff
@@ -1519,4 +1519,27 @@ public function emogrifyAutomaticallyClosesUnclosedTag()
             $this->subject->emogrify()
         );
     }
+
+    /**
+     * @test
+     */
+    public function irrelevantMediaQueriesAreRemoved()
```

Comment:
Please let's split this into two tests: one that tests that media queries with matches are not removed, and one that tests that media queries without matches are removed.

And could you also please also add one test for the case of empty media queries?